config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -10,22 +10,15 @@ presubmits:
     labels:
       preset-service-account: "true"
     optional: true
-    always_run: false
+    always_run: true
```

Comment:
I've been testing this, it behaves as well as the bazel job.
We can start running it everywhere but *not* require it, then transition which job is required, then phase out the old job.